src/core/ddsi/include/dds/ddsi/shm_transport.h

```diff
@@ -12,16 +12,17 @@
 #ifndef DDS_SHM__TRANSPORT_H
```

Comment:
I'd prefer renaming it to `ddsi_shm_transport.h`. (Just like I fully intend to one day sit down and rename all the other files in that directory that don't start with `ddsi_`).

Reply:
Ok, at some point I thought there should be a separate module shm or similar, which builds upon ddsi and can be used to extend ddsi with shared memory support. This requires quite some refactoring (also getting rid ofmost of  the #ifdefs...) so is not  going to happen here.